packages/patternfly-4/react-core/src/components/Nav/NavListItem.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Nav/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** Content rendered inside the list item */
+  children: PropTypes.node.isRequired,
+  /** Additional classes added to the list item */
+  className: PropTypes.string,
+  /** Target navigation link */
+  to: PropTypes.string,
+  /** Flag indicating whether the item is active */
+  active: PropTypes.bool
+};
+
+const defaultProps = {
+  className: '',
+  to: '#',
+  active: false
+};
+
+/* eslint react/prop-types: 0 */
+class NavListItem extends React.Component {
+  onUpdate(e) {
+    const { updateActive, groupId, id } = this.props;
+    if (e) {
+      e.stopPropagation();
+    }
+    // Callback to Nav
+    updateActive(groupId, id);
```

Comment:
should we add updateActive to the props?

Reply:
No since it is only exposed on the `Nav` component